tests/functional-tests/src/test/groovy/pages/app/ComfortHomePage.groovy

```diff
@@ -0,0 +1,22 @@
+package pages
+
+class ComfortHomePage extends BaseAppPage {
+
+    static at = { title == 'Common Forms Toolkit' }
+    static url = 'home'
+
+    static content = {
+        toolbar_Title { $(".v-toolbar__title.title", text: contains('Common Forms Toolkit')) }
+        toolbar_Login { $("button",0) }
+        screen_Login { $("button",1) }
+        
+    }
```

Comment:
I'm noticing that certain groovy files are indented with 2 spaces, whereas others are either 4 spaces or with tabs. This should be made consistent to one standard and added to the editorconfig top level file.

Reply:
Don't see the point.

Reply:
https://github.com/bcgov/nr-get-token/wiki/Coding-Style-Guide#groovy